### ISSUE TITLE
Issue with `end` parameter

### ISSUE THREAD

**Issue by darrenburns** [willmcgugan/rich#12]
I'm looking for a way to write something to my terminal without automatically taking a newline. With the `print` builtin, I can pass an empty string as the `end` paramter. The `end` parameter in `Console.print` doesn't seem to work in the same way however.

**Example:**
```python
print(".", end="")
print(".", end="")
print(".", end="")

console.print(".", end="")
console.print(".", end="")
console.print(".", end="")
```

**Output:**
```
....
.
.
```

**Expected output:**
```
......
```

It also seems to add a space before the `end` character:

**Example:**
```python
print(".", end="x")
console.print(".", end="x")
```

**Output:**
```
.x. x
```

**Expected output:**
```
.x.x
```

**Comment by willmcgugan** [comment]
It should work in the same way as built in print, so that looks like a bug.

In general though, you may be better off appending to a single Text instance then printing that. It may break wrapping if the cursor isn’t at the beginning of a line.

**Comment by darrenburns** [comment]
> In general though, you may be better off appending to a single Text instance then printing that. It may break wrapping if the cursor isn’t at the beginning of a line.

I might be misunderstanding you here, but I don't want to print the full line in one go. I'd like to print a character at a time. The dots represent test results, and will be printed as they come in.

**Comment by willmcgugan** [comment]
Ah, I see. In that case, multiple prints makes sense.

**Comment by willmcgugan** [comment]
Should be fixed in v0.6.0.
